Fix benchmark module

Trying to run the task `:swirlds-benchmarks:jmh` results in the exception below. The fix is to register the converter in every module that includes a wiring configuration, so that the modules can be used independently.

```
Exception in thread "main" java.lang.IllegalStateException: Can not create config data for record type 'class org.hiero.consensus.gossip.config.GossipWiringConfig'
	at com.swirlds.config.impl.internal.ConfigDataService.lambda$init$0(ConfigDataService.java:65)
	at java.base/java.util.concurrent.ConcurrentLinkedQueue.forEachFrom(ConcurrentLinkedQueue.java:1037)
	at java.base/java.util.concurrent.ConcurrentLinkedQueue$CLQSpliterator.forEachRemaining(ConcurrentLinkedQueue.java:894)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762)
	at com.swirlds.config.impl.internal.ConfigDataService.init(ConfigDataService.java:60)
	at com.swirlds.config.impl.internal.ConfigurationImpl.init(ConfigurationImpl.java:239)
	at com.swirlds.config.impl.internal.ConfigurationBuilderImpl.build(ConfigurationBuilderImpl.java:96)
	at com.swirlds.benchmark.BaseBench.loadConfig(BaseBench.java:81)
	at com.swirlds.benchmark.BaseBench.setup(BaseBench.java:92)
	at com.swirlds.benchmark.CryptoBench.main(CryptoBench.java:443)
Caused by: java.lang.IllegalArgumentException: No converter defined for type 'class com.swirlds.component.framework.schedulers.builders.TaskSchedulerConfiguration'
	at com.swirlds.config.impl.internal.ConverterService.convert(ConverterService.java:101)
	at com.swirlds.config.impl.internal.ConfigDataFactory.getDefaultValue(ConfigDataFactory.java:190)
	at com.swirlds.config.impl.internal.ConfigDataFactory.getValueForRecordComponent(ConfigDataFactory.java:89)
	at com.swirlds.config.impl.internal.ConfigDataFactory.lambda$createConfigInstance$0(ConfigDataFactory.java:68)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1024)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
	at com.swirlds.config.impl.internal.ConfigDataFactory.createConfigInstance(ConfigDataFactory.java:69)
	at com.swirlds.config.impl.internal.ConfigDataService.lambda$init$0(ConfigDataService.java:62)
	... 9 more
```

--- a/platform-sdk/consensus-event-creator/src/main/java/org/hiero/consensus/event/creator/config/EventCreatorConfigurationExtension.java
+++ b/platform-sdk/consensus-event-creator/src/main/java/org/hiero/consensus/event/creator/config/EventCreatorConfigurationExtension.java
@@ -22,8 +22,8 @@ public class EventCreatorConfigurationExtension implements ConfigurationExtensio
     /**
      * {@inheritDoc}
      */
-    @NonNull
     @Override
+    @NonNull
     public Set<ConverterPair<?>> getConverters() {
         return Set.of(new ConverterPair<>(TaskSchedulerConfiguration.class, TaskSchedulerConfiguration::parse));
     }

--- a/platform-sdk/consensus-event-intake/src/main/java/org/hiero/consensus/event/intake/config/EventIntakeConfigurationExtension.java
+++ b/platform-sdk/consensus-event-intake/src/main/java/org/hiero/consensus/event/intake/config/EventIntakeConfigurationExtension.java
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.hiero.consensus.event.intake.config;
 
+import com.swirlds.component.framework.schedulers.builders.TaskSchedulerConfiguration;
 import com.swirlds.config.api.ConfigurationExtension;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.util.Set;
@@ -16,5 +17,14 @@ public class EventIntakeConfigurationExtension implements ConfigurationExtension
     @NonNull
     public Set<Class<? extends Record>> getConfigDataTypes() {
         return Set.of(EventIntakeWiringConfig.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public Set<ConverterPair<?>> getConverters() {
+        return Set.of(new ConverterPair<>(TaskSchedulerConfiguration.class, TaskSchedulerConfiguration::parse));
     }
 }

--- a/platform-sdk/consensus-gossip/src/main/java/org/hiero/consensus/gossip/config/GossipConfigurationExtension.java
+++ b/platform-sdk/consensus-gossip/src/main/java/org/hiero/consensus/gossip/config/GossipConfigurationExtension.java
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.hiero.consensus.gossip.config;
 
+import com.swirlds.component.framework.schedulers.builders.TaskSchedulerConfiguration;
 import com.swirlds.config.api.ConfigurationExtension;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.util.Set;
@@ -30,6 +31,8 @@ public class GossipConfigurationExtension implements ConfigurationExtension {
     @Override
     @NonNull
     public Set<ConverterPair<?>> getConverters() {
-        return Set.of(new ConverterPair<>(NetworkEndpoint.class, new NetworkEndpointConverter()));
+        return Set.of(
+                new ConverterPair<>(NetworkEndpoint.class, new NetworkEndpointConverter()),
+                new ConverterPair<>(TaskSchedulerConfiguration.class, TaskSchedulerConfiguration::parse));
     }
 }

--- a/platform-sdk/consensus-hashgraph/src/main/java/org/hiero/consensus/hashgraph/config/HashgraphConfigurationExtension.java
+++ b/platform-sdk/consensus-hashgraph/src/main/java/org/hiero/consensus/hashgraph/config/HashgraphConfigurationExtension.java
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.hiero.consensus.hashgraph.config;
 
+import com.swirlds.component.framework.schedulers.builders.TaskSchedulerConfiguration;
 import com.swirlds.config.api.ConfigurationExtension;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.util.Set;
@@ -16,5 +17,14 @@ public class HashgraphConfigurationExtension implements ConfigurationExtension {
     @NonNull
     public Set<Class<? extends Record>> getConfigDataTypes() {
         return Set.of(HashgraphWiringConfig.class, ConsensusConfig.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public Set<ConverterPair<?>> getConverters() {
+        return Set.of(new ConverterPair<>(TaskSchedulerConfiguration.class, TaskSchedulerConfiguration::parse));
     }
 }

--- a/platform-sdk/consensus-pces/src/main/java/org/hiero/consensus/pces/config/PcesConfigurationExtension.java
+++ b/platform-sdk/consensus-pces/src/main/java/org/hiero/consensus/pces/config/PcesConfigurationExtension.java
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.hiero.consensus.pces.config;
 
+import com.swirlds.component.framework.schedulers.builders.TaskSchedulerConfiguration;
 import com.swirlds.config.api.ConfigurationExtension;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.util.Set;
@@ -16,5 +17,14 @@ public class PcesConfigurationExtension implements ConfigurationExtension {
     @NonNull
     public Set<Class<? extends Record>> getConfigDataTypes() {
         return Set.of(PcesConfig.class, PcesWiringConfig.class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public Set<ConverterPair<?>> getConverters() {
+        return Set.of(new ConverterPair<>(TaskSchedulerConfiguration.class, TaskSchedulerConfiguration::parse));
     }
 }

